### PR TITLE
SwiftLint integration

### PR DESCRIPTION
**Pull request by IvanKulundzic**
Added SwiftLint to the project. Fixed some SwiftLint warnings.

### CODE REVIEW

File: QuizApp/AppDelegate.swift [hunk 1]
```diff
@@ -10,16 +10,19 @@ import UIKit
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    func application(_ application: UIApplication,
```
Comment (istaudacher) [inline QuizApp/AppDelegate.swift]:
```
func application(
    _ application: UIApplication,
    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
) -> Bool {
    ...
}
```

File: QuizApp/AppDelegate.swift [hunk 2]
```diff
@@ -10,16 +10,19 @@ import UIKit
...
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         true
     }
 
     // MARK: UISceneSession Lifecycle
-    
-    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
+
+    func application(_ application: UIApplication,
```
Comment (istaudacher) [inline QuizApp/AppDelegate.swift]: as above

File: QuizApp/SceneDelegate.swift [hunk 3]
```diff
@@ -11,11 +11,13 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
     var window: UIWindow?
 
-    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+    func scene(_ scene: UIScene,
```
Comment (istaudacher) [inline QuizApp/SceneDelegate.swift]: as above

**Review by istaudacher — CHANGES_REQUESTED** [review]

File: QuizApp/AppDelegate.swift (hunk 1 above)
Comment (IvanKulundzic) [reply QuizApp/AppDelegate.swift]: Corrected.

File: QuizApp/SceneDelegate.swift (hunk 3 above)
Comment (IvanKulundzic) [reply QuizApp/SceneDelegate.swift]: Corrected.

File: QuizApp/AppDelegate.swift (hunk 2 above)
Comment (IvanKulundzic) [reply QuizApp/AppDelegate.swift]: Corrected.

**Review by istaudacher — APPROVED** [review]

**Pull request merged**